### PR TITLE
Refer to official TS performance wiki

### PR DESCRIPTION
**Pull request by elektronik2k5**

### CODE REVIEW

File: docs/tips/typescript.md [hunk 1]
```diff
@@ -64,7 +64,9 @@ interface ITodoSnapshotIn extends SnapshotIn<typeof Todo> {} // => { title?: str
 interface ITodoSnapshotOut extends SnapshotOut<typeof Todo> {} // => { title: string }
 ```
 
-Note, it is important to use `interface` and not `type` when constructing those types! Although `type`s will work exactly the same, due to their nature they will be much more expensive for the compiler to typecheck.
+Note, it is important to use `interface` and not `type` when constructing those types! Although `type`s will work exactly the same, due to their nature they will be (much more expensive for the compiler to typecheck)[https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections].
+
+For further performace tips, read the (official TypeScript performance wiki)[https://github.com/microsoft/TypeScript/wiki/Performance].
```
Comment (jamonholmgren) [inline docs/tips/typescript.md:69]:
```suggestion
For further performance tips, read the [official TypeScript performance wiki](https://github.com/microsoft/TypeScript/wiki/Performance).
```

**Review by jamonholmgren — CHANGES_REQUESTED** [review]
Just small tweaks needed.

File: docs/tips/typescript.md (hunk 1 above)
Comment (elektronik2k5) [reply docs/tips/typescript.md:69]: Oops, I swapped the braces and brackets :sweat_smile:

**Review by jamonholmgren — APPROVED** [review]
Thank you @elektronik2k5 !

**Pull request merged**